coreTable/CoreTableColumns.ts

```diff
@@ -724,6 +724,7 @@ export const ColumnTypeMap = {
     Date: DateColumn,
     Year: YearColumn,
     Quarter: QuarterColumn,
+    Time: TimeColumn,
```

Comment:
Potential issue: `TimeColumn` is `abstract`. I just exposed it to check that a column is of _any_ time instance type. I'm not sure if there are some negative consequences...

Reply:
Hm, the only bad side-effect that I can think is that you could then include a `TimeColumn` in a spreadsheet configuration.

Reply:
Right. TS abstract classes become JS classes, so at runtime it doesn't matter that it's abstract. It's kind of like `NumericColumn`.